doc/instore-app/printer-guide/printer.md

```diff
@@ -68,6 +67,9 @@ configure it within the fiskaltrust InStore App.
 3. In the Printer Settings section, scroll down to find **Printers**.
 4. Select the type of printer you have (Bluetooth, Wi-Fi, or USB).
 5. Choose your printer from the list of available devices.
```

Comment:
This feels a little confusing because line 4 and 5 seem to mean the same. And 3 should mention to also click on  printers and not only search for it.

```suggestion
3. Scroll down to find the **Printer Settings** section and click on **Printers** to open the list of available printer options.
4. Select the type of printer you want to use from the list.
```